.ci/Jenkinsfile

```
@@ -581,11 +585,21 @@ def generateFunctionalTestStep(Map args = [:]){
                 runId,
                 "-t destroy --extra-vars=\"nodeLabel=${platform} nodeUser=${machine.username} nodeImage=${machine.image} nodeInstanceType=${machine.instance_type}\"")
         }
+        withEnv([
+          "ARCHITECTURE=${goArch}",
+          "CUCUMBER_REPORTS_PATH=${BASE_DIR}/outputs/${testRunner.ip}",
```

My guess, the base_dir is not needed since it's already within the base_dir context, in addition L568 creates the folder structure (https://github.com/elastic/e2e-testing/pull/2245/files#diff-ef8b2f5e4d21d81c0eba4754dea3a3dfc2c67d26284f195165b650cf4aff7beaR568)

```suggestion
          "CUCUMBER_REPORTS_PATH=outputs/${testRunner.ip}",
```

---

Yes, you are right. I removed the dir(BASE_DIR) but did not update that variable. Good catch!